app-libs/stf/src/helpers.rs

```diff
@@ -141,6 +148,21 @@ pub fn get_parentchain_number() -> Option<BlockNumber> {
 	get_storage_value("Parentchain", "Number")
 }
 
+/// Ensures an account is a registered enclave account.
+pub fn ensure_registered_enclave(account: &AccountId) -> StfResult<()> {
```

Comment:
I think this name is a little misleading. This check does not have anything to do with `regsitered`, but rather if shares the same state  / shielding key.

Suggestion:
`ensure_enclave_signer_account`

Reply:
Good input 👍